src/node/hooks/express/openapi.js

```diff
@@ -596,9 +595,13 @@ exports.expressPreSession = async (hookName, {app}) => {
           // read form data if method was POST
           let formData = {};
           if (c.request.method === 'post') {
-            const form = new formidable.IncomingForm();
-            const parseForm = promisify(form.parse).bind(form);
-            formData = await parseForm(req);
+            const form = new IncomingForm();
+            formData = (await form.parse(req))[0];
+            for (const k of Object.keys(formData)) {
+              if (formData[k] instanceof Array) {
```

Comment:
formidable also exports a helper function firstValues

Reply:
Where should that method be called upon? Line 599? I tried firstValues(await form.parse(req)) but that didn't work.

Reply:
after you have fields
            formData = (await form.parse(req))[0]; //[0] is fields [1] is files
            const formDataSingle = firstValues(form, formData);

Reply:
and then use formDataSingle variable instead of formData

Reply:
I put it like that. But it is failing. 

```javascript
          let formDataSingle;
          // read form data if method was POST
          if (c.request.method === 'post') {
            let formData = {};
            const form = new IncomingForm();
            formData = (await form.parse(req))[0];
            formDataSingle = firstValues(form, formData);
          }
```

Reply:
What is failing exactly ?
In 3 days I will have a bit of time maybe I will try to see what is wrong with this PR by trying out locally

Reply:
Thanks for the help. That was the run: https://github.com/ether/etherpad-lite/actions/runs/5672397436/job/15371523032?pr=5796 .  I turned on debug level but there is still only the diff of the texts. 
So it seems like there is not a lot of error handling/output in that corner of etherpad. 

I noticed that Linux/Darwin + NPM6 works best for running etherpad. Windows is kind of buggy.

Reply:
@GrosSacASac Could you find some time to check why your suggestion with firstValue doesn't work? It might be interesting to learn why your suggestions seems to not work.

Reply:
I am on a trip now